auraed/src/runtime/mod.rs

```diff
@@ -103,26 +97,25 @@ impl CellService {
             // Final Build
             .build(hierarchy);
 
-        let mut cgroup_cache =
-            self.cgroup_table.lock().expect("lock cgroup_table");
-        // Check if there was already a cgroup in the table with this cell name as a key.
-        if let Some(_old_cgroup) =
-            cgroup_cache.insert(cell_name.into(), cgroup.clone())
-        {
-            return Err(anyhow!("cgroup already exists for {cell_name}"));
-        };
+        self.cgroup_table.insert(cell_name, &cgroup).map_err(|e| {
```

Comment:
What do we do if the cgroup already exists? Should we error here?

Reply:
we do.  it's in the table implementation.